### DISCUSSION TITLE
how to prevent last opened file or workspace

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#13783] [Q&A]
hi,
   When I open an workspace  or files in theia, and I close theia, next time I start theia, it will open the workspace and files last time I opened. Is there any option can prevent theia remember last opened workspace and files, I want to keep empty everytime I start theia

**Comment by msujew** [comment]
Hey @wss29,

on shutdown, Theia simply stores all widgets in the localstorage and then restores them on load. You can probably override the `ShellLayoutRestorer` to prevent any data from being stored/restored.

**Reply by wss29** [reply]
thanks a great
